test/resource/http.js

```diff
@@ -93,7 +92,7 @@ exports.server = function _listen(ready) {
 
     req.on('data', (chunk) => chunks.push(chunk));
     req.on('end', (chunk) => {
-      if (chunk) chunks.push(chunk);
+      if (chunk) { chunks.push(chunk); }
```

Comment:
It's this addition... https://coveralls.io/builds/8416252/source?filename=lib%2Fsignature.js#L25 Somehow 3 lines account for 1.5 percent of the project 😒